src/actions/action.surrounder.ts

```diff
@@ -0,0 +1,72 @@
+import { Context } from "telegraf";
+import { Message, User } from "telegraf/typings/core/types/typegram";
+import { AxiosError } from "axios";
+
+export default abstract class Action {
+  id: string;
+  message: Message.CommonMessage;
+  inline_message_id: string;
+  from: User;
+  item_name: string;
+  context: Context;
+
+  get ctx() {
+    return this.context;
+  }
+
+  set ctx (ctx: Context) {
+    this.context = ctx;
+    this.id = (ctx.callbackQuery as any).data.split(':')[1];
+    this.message = ctx.callbackQuery.message;
+    this.inline_message_id = ctx.callbackQuery.inline_message_id;
+    this.from = ctx.callbackQuery.from;
+  }
+
+  async execute(): Promise<void> {
+    await this.updateState();
+
+    const extra: object = {
```

Comment:
```suggestion
    const markup: object = {
```

Reply:
the name of a variable `extra` follows from 
![image](https://user-images.githubusercontent.com/42845566/178159135-c2faa446-9e4e-469d-a1ad-d58d72b63956.png)
where markup is a **field** of the object `extra`